sys-utils/renice.1.adoc

```diff
@@ -46,16 +46,22 @@ renice - alter priority of running processes
 
 == SYNOPSIS
 
-*renice* [*-n*] _priority_ [*-g*|*-p*|*-u*] _identifier_...
+*renice* [*-n|--priority|--relative*] _priority_ [*-g*|*-p*|*-u*] _identifier_...
```

Comment:
What's the point of `--priority`? It also seems a bit inconsistent wrt `--relative`.

Reply:
Well, `--priority` was already there before and I left it as it was as I didn't want to break backwards compatibility. I just added `--relative`, should anyone want to use the "relative" version without setting up an environment var. Do you mind to explain a bit more what I need to do? Thanks.

Reply:
I didnt't realize tht `--priority` alrrady exists. Please disregard this comment.

Reply:
Updated everything following your suggestions and squashed all commits.